trantor/utils/Utilities.cc:22:10: fatal error: xlocale.h: No such file or directory

## Error:

```
ERROR: /home/username/.cache/bazel/_bazel_username/73dfd4fd85663bb6f1bb50c53ee60ba3/external/trantor/BUILD.bazel:109:11: Compiling trantor/utils/Utilities.cc failed: (Exit 1): gcc failed: error executing command
  (cd /home/username/.cache/bazel/_bazel_username/73dfd4fd85663bb6f1bb50c53ee60ba3/execroot/foo && \
...
    PWD=/proc/self/cwd \
  /usr/bin/gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer '-std=c++0x' -MD -MF bazel-out/k8-fastbuild/bin/external/trantor/_objs/trantor/Utilities.pic.d '-frandom-seed=bazel-out/k8-fastbuild/bin/external/trantor/_objs/trantor/Utilities.pic.o' -fPIC -iquote external/trantor -iquote bazel-out/k8-fastbuild/bin/external/trantor -iquote external/boringssl -iquote bazel-out/k8-fastbuild/bin/external/boringssl -Ibazel-out/k8-fastbuild/bin/external/trantor/_virtual_includes/trantor -isystem external/trantor/trantor -isystem bazel-out/k8-fastbuild/bin/external/trantor/trantor -isystem external/trantor/trantor/utils -isystem bazel-out/k8-fastbuild/bin/external/trantor/trantor/utils -isystem external/trantor/trantor/net -isystem bazel-out/k8-fastbuild/bin/external/trantor/trantor/net -isystem external/trantor/trantor/net/inner -isystem bazel-out/k8-fastbuild/bin/external/trantor/trantor/net/inner -isystem external/boringssl/src/include -isystem bazel-out/k8-fastbuild/bin/external/boringssl/src/include -isystem bazel-out/k8-fastbuild/bin/external/cares/libcares/include -DHAVE_BAZEL_BUILD '-std=c++2a' -Wall -Wreturn-type -Wuninitialized -Wunused-result '-Werror=narrowing' '-Werror=reorder' -Wunused-local-typedefs '-Werror=conversion-null' '-Werror=overlength-strings' '-Werror=pointer-arith' '-Werror=varargs' '-Werror=vla' '-Werror=write-strings' -Wmissing-declarations '-fdiagnostics-color=always' '-std=c++17' -Wconversion -fno-canonical-system-headers -Wno-builtin-macro-redefined '-D__DATE__="redacted"' '-D__TIMESTAMP__="redacted"' '-D__TIME__="redacted"' -c external/trantor/trantor/utils/Utilities.cc -o bazel-out/k8-fastbuild/bin/external/trantor/_objs/trantor/Utilities.pic.o)
Execution platform: @local_config_platform//:host
external/trantor/trantor/utils/Utilities.cc:22:10: fatal error: xlocale.h: No such file or directory
   22 | #include <xlocale.h>
      |          ^~~~~~~~~~~
compilation terminated.
INFO: Elapsed time: 3.431s, Critical Path: 0.12s
INFO: 17 processes: 17 internal.
FAILED: Build did NOT complete successfully

```
## OS and compiler information:
❯ uname -a
Linux mustang 5.10.43.3-microsoft-standard-WSL2 #1 SMP Wed Jun 16 23:47:55 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
❯ gcc --version
gcc (Ubuntu 10.3.0-1ubuntu1) 10.3.0
Copyright (C) 2020 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

## Probable cause:

I believe the header has been removed.
https://sourceware.org/glibc/wiki/Release/2.26#Removal_of_.27xlocale.h.27

--- a/trantor/utils/Utilities.cc
+++ b/trantor/utils/Utilities.cc
@@ -19,7 +19,7 @@
 #else  // _WIN32
 #if __cplusplus < 201103L || __cplusplus >= 201703L
 #include <stdlib.h>
-#include <xlocale.h>
+#include <locale.h>
 #else  // __cplusplus
 #include <locale>
 #include <codecvt>

